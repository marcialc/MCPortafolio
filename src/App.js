import React, { Component } from "react";
import Home from "./screens/Home";
import About from "./screens/About";
import Experience from "./screens/Experience";

class App extends Component {
  render() {
    return (
      <div className="App">
        <Home />
        <About />
        <Experience />
      </div>
    );
  }
}

export default App;
