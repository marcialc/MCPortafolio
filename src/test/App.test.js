import React from 'react';
import App from "../App";
import Adapter from 'enzyme-adapter-react-16';
import renderer from 'react-test-renderer';
import { shallow, configure } from 'enzyme';

configure({adapter: new Adapter()});

describe('React component test with Enzyme', () => {
  it('renders home crashing', () => {
     shallow(<App></App>);
   });
});



