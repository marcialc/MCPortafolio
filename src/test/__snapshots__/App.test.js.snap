// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="App"
>
  <header
    id="home"
  >
    <nav
      id="nav-wrap"
    >
      <ul
        className="nav"
        id="nav"
      >
        <li
          className="current"
        >
          <a
            className="smoothscroll"
            href="#home"
          >
            Home
          </a>
        </li>
        <li>
          <a
            className="smoothscroll"
            href="#about"
          >
            About
          </a>
        </li>
        <li>
          <a
            className="smoothscroll"
            href="#experience"
          >
            Experience
          </a>
        </li>
      </ul>
    </nav>
    <div
      className="row banner"
    >
      <div
        className="banner-text"
      >
        <h1
          className="responsive-headline"
        >
          Marcial Cabrera
        </h1>
        <h3>
          Software Engineer / Front-end Developer / Mobile Developer
        </h3>
        <ul
          className="social"
        >
          <li>
            <a
              href="https://www.linkedin.com/in/marcialc"
            >
              <i
                className="fa fa-linkedin"
              />
            </a>
          </li>
          <li>
            <a
              href="http://github.com/marcialc"
            >
              <i
                className="fa fa-github"
              />
            </a>
          </li>
        </ul>
      </div>
    </div>
  </header>
  <section
    id="about"
  >
    <div
      className="row"
    >
      <div
        className="three columns"
      >
        <img
          alt=""
          className="profile-pic"
          src="assets/me2.jpg"
        />
      </div>
      <div
        className="nine columns main-col"
      >
        <h2>
          About Me
        </h2>
        <p>
          Currently a Senior in Florida International University (FIU) Pursing a Bachelors of Science in Computer Science. Very passionated about about my field and whats to come and the new tehcnologies and challenges it brings.
        </p>
        <div
          className="row"
        >
          <div
            className="columns contact-details"
          >
            <h2>
              Contact
            </h2>
            <p
              className="address"
            >
              <span>
                Marcial Cabrera
              </span>
              <br />
              <span>
                Miami, FL
              </span>
              <br />
              <span>
                marcialandres06@gmail.com
              </span>
            </p>
          </div>
        </div>
      </div>
    </div>
  </section>
  <section
    id="experience"
  >
    <div
      className="row education"
    >
      <div
        className="three columns header-col"
      >
        <h1>
          <span>
            education
          </span>
        </h1>
      </div>
      <div
        className="nine columns main-col"
      >
        <div>
          <h2>
            Florida International University (FIU)
          </h2>
          <p
            className="major"
          >
            B.S Computer Science
            <br />
            <p
              className="date"
            >
              Expected Graduation Date: Fall 2019
            </p>
          </p>
        </div>
      </div>
    </div>
    <div
      className="row work"
    >
      <div
        className="three columns header-col"
      >
        <h1>
          <span>
            work
          </span>
        </h1>
      </div>
      <div
        className="nine columns main-col"
      >
        <div
          className="row item"
        >
          <div
            className="twelve columns"
          >
            <h2>
              American Express
               
              <em
                style={
                  Object {
                    "fontSize": 12,
                    "paddingLeft": 30,
                  }
                }
              >
                June 2019
                 - 
                August 2019
              </em>
            </h2>
            <p
              className="title"
            >
              Mobile Engineer Intern
            </p>
            <p
              className="date"
            >
               
              Conducted code reviews and tested ongoin sprints while being a core memeber of a Agile team. Designed and developed on existing movile application on multiple channels such as React Native, Android and Java services/APIs.
            </p>
          </div>
        </div>
        <div
          className="row item"
        >
          <div
            className="twelve columns"
          >
            <h2>
              CodeArt
               
              <em
                style={
                  Object {
                    "fontSize": 12,
                    "paddingLeft": 30,
                  }
                }
              >
                June 2019
                 - 
                Present
              </em>
            </h2>
            <p
              className="title"
            >
              Programming Teacher
            </p>
            <p
              className="date"
            >
               
              As an instructor at CodeArt, my goals are to help the cause that CodeArt is continuing to support by opening the doors to the previously male-dominated field of coding, CodeArt inspires young women to explore coding, showcasing their talents, and pursue careers in this growing industry.
            </p>
          </div>
        </div>
      </div>
    </div>
    <div
      className="row skill"
    >
      <div
        className="three columns header-col"
      >
        <h1>
          <span>
            Programming
            <p />
            Languages
          </span>
        </h1>
      </div>
      <div
        className="nine columns main-col"
      >
        <p>
          
        </p>
        <div
          className="bars"
        >
          <ul
            className="programmminglanguages"
          >
            <li>
              <span
                className="bar-expand java"
              />
              <em>
                JAVA
              </em>
            </li>
            <li>
              <span
                className="bar-expand swift"
              />
              <em>
                SWIFT
              </em>
            </li>
            <li>
              <span
                className="bar-expand typescript"
              />
              <em>
                TypeScript
              </em>
            </li>
            <li>
              <span
                className="bar-expand javascript"
              />
              <em>
                JavaScript
              </em>
            </li>
            <li>
              <span
                className="bar-expand kotlin"
              />
              <em>
                Kotlin
              </em>
            </li>
          </ul>
        </div>
      </div>
    </div>
  </section>
</div>
`;
